torch/_functorch/aot_autograd.py

```diff
@@ -551,6 +551,16 @@ def __post_init__(self):
             o.dynamic_dims for o in self.output_info
         )
 
+        self.is_rng_op_functionalized = config.functionalize_rng_ops
```

Comment:
It's a bit strange that you have to capture the config setting in the collected metadata

Reply:
Is this because the config setting may change later? I guess that makes sense.

Reply:
Yes, its because config setting can change later.